### DISCUSSION TITLE
新功能投票

### DISCUSSION THREAD

**Discussion by whsheng** [tw93/MiaoYan#248] [Ideas]
非常喜欢「**妙言**」这款产品，界面美观 、功能简单好用，作者更新很频繁。自己使用过程中，希望能有如下功能：

1. 导出的PDF支持分页
2. 导出的文件支持加水印
3. 支持tag或目录分级，便于文件快速查找
4. 支持对指定文件加密访问
5. 支持对指定文件的历史版本管理
6. 支持一键呼出「快捷键/帮助」，感觉妙言的功能很简单，但很多小技巧不知道怎么使用？

**Comment by tw93** [comment]
赞赞赞
3、4、5、6感觉都是好功能
1、2 需要看默认mac库是否支持
我这边逐步来安排

**Comment by luo-h** [comment]
希望增加账号体系，资源存储，换电脑不好迁移啦

**Comment by kyranhsiao** [comment]
导出的PDF分页功能挺需要的，虽然可以用其他工具把md文件分页，直接打印，但是妙言的字体和布局很好看，很需要这个功能！
